guide/content/en/guide/basics/sync_async.md

```diff
@@ -0,0 +1,175 @@
+
+To illustrate this, the simplest possible example is used below:
+
+Ex. 3
+```py test3.py
+#!/usr/bin/env python3.11
+import asyncio
+from random import randrange
+
+async def async_print(text: str) -> None:
+    rand_wait: int = randrange(0,5)
+    await asyncio.sleep(rand_wait)
+    print(text)
+
+async def main() -> None:
+    my_name: str = 'Sanic'
+    await async_print('I go first')
+    await async_print(f'Hello {my_name}, I run second')
+    await async_print('I go third')
+
+asyncio.run(main())
+```
+
+.. tip:: Note
+
+    💡 Like the use of *async* and *await*, some of the complexity here is deliberately avoided for the sake of a higher level understanding.
+
+Immediately there's much more going on here - we are importing the **asyncio** standard library and there are two new keywords in play, **async** and **await**. It should be noted that asynchronous execution has come quite far since it was first introduced, and much more is done under the hood for the sake of readability and reusability, but in my opinion it masks the inner workings of asynchronous code and prevents people from understanding why it works as well as it does.
+
+In the example, instead of calling print directly, we use the async keyword to create a *coroutine* called **async_print** that takes a single text argument.
+
+Also in the example **main** is now a coroutine as well. This is because coroutines can easily schedule other coroutines and call regular functions, but regular functions cannot easily schedule coroutines.
+
+.. tip:: Note
+
+    💡 It is not impossible for regular functions to schedule coroutines, but it is beyond the scope of this guide.
+
+Scheduling has been referenced several times at this point, and it should be discussed.
+
+There are as many good articles about async/await in python as there are poorly written ones, but one that appears high in search results states that ***await*** pauses execution. That doesn't make sense, though, does it? What would be the point in telling a function that it should just not do anything.
+
+However this guide is not going to dig into the material impact of scheduling, but while analogies in describing async/await are always suspect, consider this example scenario:
+
+You wish to brew a cup of tea.
+
+Using an electric kettle, you fill it with water, and start the kettle.
+
+Most human beings are not that interested in standing at the kettle, doing nothing until it boils, but with regular functions, that's what happens: Fill kettle. Boil. Use water to make drink. The operations happen in order, and you cannot do anything else until they complete.
+
+For someone making a drink, that seems like a waste of time. A person does not need to stand around and watch the kettle until it boils, instead, other things can be done. So the kettle is left to do its task, which by filling it and turning it on, we can consider to be scheduled, and when it is done, we can come back to it and make our hot drink.
```

Comment:
I know this is trying to talk to a wide audience but I feel its too dumbed down to be of use to someone that actually gets to the point of reading it. i think that most people that would be reading this would definitely have an understanding of `asyncio` (though the motivation here could be something I've just not witnessed!)

I think you could deal with the concept in far fewer lines and do away with a noddy example, and defer more detail to https://docs.python.org/3/library/asyncio.html and go on to deal with `sanic` specific detail instead ?